build.sbt

```diff
@@ -131,6 +131,7 @@ lazy val allAggregates = core.projectRefs ++
   playServer.projectRefs ++
   vertxServer.projectRefs ++
   zioServer.projectRefs ++
```

Comment:
the `zioServer` name is misleading now. Maybe we could rename it to `zioHttp4sServer`?